README.md

```diff
@@ -8,6 +8,16 @@ Yarner works with familiar syntax, which can be further customized to suit your
 See the [examples](examples) directory for full working examples.
 See the [**User Guide**](https://mlange-42.github.io/yarner/) for documentation.
 
+## Features
+
+* Macros and named code blocks
+* Multiple files, multiple entrypoints
+* File transclusions
+* Reverse mode
+* ...
+
+See the [**User Guide**](https://mlange-42.github.io/yarner/) for a complete and detailed explanation of all features.
```

Comment:
I wonder whether it would make sense to slim down the README now that the book exists? But admittedly from a first glance, I have no section which I would be comfortable with removing...

Reply:
I already reduced it a bit. But I would still leave the basics here: installation, first steps/getting started, and an illustrative example. Maybe the entrypoints section, but I am not really sure about. 

Maybe we should let it read someone completely new to Yarner.